packages/python-sqlparse/python-sqlparse.spec

```diff
@@ -6,7 +6,7 @@
 
 Name:           %{?scl_prefix}python-%{pypi_name}
 Version:        0.4.2
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
please add a changelog entry for that one :)

Reply:
Done, my bad